craft_providers/actions/snap_installer.py

```diff
@@ -111,6 +149,21 @@ def inject_from_host(*, executor: Executor, snap_name: str, classic: bool) -> No
     :raises SnapInstallationError: on unexpected error.
     """
     target_snap_path = pathlib.Path(f"/tmp/{snap_name}.snap")
+    host_revision = _get_host_snap_revision(snap_name=snap_name)
+    target_revision = _get_target_snap_revision(snap_name=snap_name, executor=executor)
+    logger.debug(
+        "For snap %r: host revision = %r, target revision = %r",
```

Comment:
nitpick (IMO shorter and more clear): "Revisions found for snap %r: host=%r, target=%r"